target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -27,6 +27,9 @@ ipq40xx_setup_interfaces()
 	zyxel,nbg6617)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
 		;;
+	linksys,whw03v2)
+		ucidef_set_interfaces_lan_wan "lan" "wan"
```

Comment:
There are already devices using the same config, just add this one there

Reply:
Sure, my bad, I changed this config a couple of time and forgot to double check.

Reply:
Done